pkg/util/convert/convert.go

```diff
@@ -0,0 +1,87 @@
+package convert
+
+import (
+	"fmt"
+
+	"sigs.k8s.io/yaml"
+
+	kudov1beta1 "github.com/kudobuilder/kudo/pkg/apis/kudo/v1beta1"
+)
+
+// String returns a pointer to the string value passed in.
+func String(v string) *string {
+	return &v
+}
+
+// StringValue returns the value of the string pointer passed in or
+// "" if the pointer is nil.
+func StringValue(v *string) string {
+	if v != nil {
+		return *v
+	}
+	return ""
+}
+
+// UnwrapParamValue unwraps a parameter value.
```

Comment:
the docs repeating the name doesn't help here... can we eliminate "unwraps a parameter value" and replace with line below?